<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MergeDialog</class>
 <widget class="QDialog" name="MergeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListView" name="m_fileListView">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>361</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>120</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Data delimiter:</string>
   </property>
   <widget class="QCheckBox" name="m_commaCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>,Comma</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="m_semiCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>;Semicolon</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="m_spaceCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Space</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="m_tabCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tab</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="m_otherEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_otherCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Other</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>211</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Position setting</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>head row</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="m_headrowEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>42</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>data start row</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="m_dataStartRowEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>42</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X colmun</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="m_XColumnEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>42</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="m_YStartColumnEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>42</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y start colmun</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="m_stepEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>42</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Y colmun step</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="m_offsetDataHeadEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>42</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-20</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>data col. offset from head</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="m_selectButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>390</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_saveButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>390</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_quitButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Merging by</string>
   </property>
   <widget class="QCheckBox" name="m_colmunSpanCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>89</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>column span</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="m_rowSpanCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>89</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>row span</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>En/Decoding Style</string>
   </property>
   <widget class="QComboBox" name="m_srcEncodingCBBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Source Files:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="m_destEncodingCBBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Target Files:</string>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>54</width>
     <height>71</height>
    </rect>
   </property>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Index setting</string>
   </property>
   <widget class="QCheckBox" name="m_XasIndexCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X col. as index</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_XisDateTimeCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X col. is DateTime Type</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="m_datetimeFormatCBBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>60</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DateTime Format:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
